.azure-pipelines/test_plan.py

```diff
@@ -466,6 +482,66 @@ def poll(self, test_plan_id, interval=60, timeout=-1, expected_state=""):
         required=False,
         help="Repository name from Azure Pipelines"
     )
+    parser_create.add_argument(
+        "--testbed-name",
+        type=str,
+        dest="testbed_name",
+        nargs='?',
+        const='',
+        default=None,
+        required=False,
+        help="Testbed name, Split by ',', like: testbed1,testbed2"
+    )
+    parser_create.add_argument(
+        "--image_url",
+        type=str,
+        dest="image_url",
+        nargs='?',
+        const='',
+        default=None,
+        required=False,
+        help="Image url"
+    )
+    parser_create.add_argument(
+        "--hwsku",
+        type=str,
+        dest="hwsku",
+        nargs='?',
+        const='',
+        default=None,
+        required=False,
+        help="Hardware SKU."
+    )
+    parser_create.add_argument(
+        "--test-plan-type",
+        type=str,
+        dest="test_plan_type",
+        nargs='?',
+        const='',
+        default="PR",
+        required=False,
```

Comment:
Add an argument like `choices` to limit the allowed test plan types?

Reply:
Refined